x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts

```diff
@@ -64,18 +64,19 @@ async function asyncSearch(
   options?: ISearchOptions
 ) {
   const { timeout = undefined, restTotalHitsAsInt = undefined, ...params } = {
-    trackTotalHits: true, // Get the exact count of hits
     ...request.params,
   };
 
+  params.trackTotalHits = true; // Get the exact count of hits
+
   // If we have an ID, then just poll for that ID, otherwise send the entire request body
   const { body = undefined, index = undefined, ...queryParams } = request.id ? {} : params;
 
   const method = request.id ? 'GET' : 'POST';
   const path = encodeURI(request.id ? `/_async_search/${request.id}` : `/${index}/_async_search`);
 
   // Wait up to 1s for the response to return
-  const query = toSnakeCase({ waitForCompletionTimeout: '1s', ...queryParams });
+  const query = toSnakeCase({ waitForCompletionTimeout: '100ms', ...queryParams });
```

Comment:
Is this intended change? not sure, because isn't mentioned in pr description.

Reply:
Yes. It's intentional.
I will document it in the PR.